### PR TITLE
rhit 1.5.3 (new formula)

### LINKED ISSUE #9: Homebrew support and request github release
👋  we have a pending PR of including this utility into the homebrew-core, it would be great that we can do some github release managements, so that the downstream packaging would be beneficial from it. Thanks!

relates to:
- https://github.com/Homebrew/homebrew-core/pull/81076

### PR DESCRIPTION
**Pull request by vladimyr**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

---

Closes https://github.com/Canop/rhit/issues/9

### CODE REVIEW

File: Formula/rhit.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
+class Rhit < Formula
+  desc "Nginx log explorer"
+  homepage "https://github.com/Canop/rhit"
```
Comment (chenrui333) [inline Formula/rhit.rb:3]:
```suggestion
  homepage "https://dystroy.org/rhit/"
```

File: Formula/rhit.rb [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  url "https://static.crates.io/crates/rhit/rhit-1.5.2.crate"
```
Comment (chenrui333) [inline Formula/rhit.rb:4]: crate file seems pretty intensive in here, maybe we should ask upstream to create a new release and maintain it?

Comment (vladimyr) [reply Formula/rhit.rb:4]: Actually, @Canop does regular releases but only in binary form (https://dystroy.org/rhit/download/), and the git repo doesn't have tags. 🤷

Comment (chenrui333) [reply Formula/rhit.rb:4]: ok, we can use the tgz file now. 🎉

File: Formula/rhit.rb [hunk 3]
```diff
@@ -0,0 +1,50 @@
+class Rhit < Formula
+  desc "Nginx log explorer"
+  homepage "https://dystroy.org/rhit/"
+  url "https://github.com/Canop/rhit/archive/refs/tags/v1.5.3.tar.gz"
+  sha256 "fc2bd613a9ded70906f4c2af67a1540ffe6de165efe6aefeffaea97ceed76a82"
+  license "MIT"
+  head "https://github.com/Canop/rhit.git"
```
Comment (nandahkrishna) [inline Formula/rhit.rb:7]:
```suggestion
  head "https://github.com/Canop/rhit.git", branch: "main"
```

Comment (vladimyr) [reply Formula/rhit.rb:7]: I'm gonna write an audit for this thing...

**Review by nandahkrishna — APPROVED** [review]
Thanks again, @vladimyr! 🍻

**Pull request closed without merging**